### ISSUE TITLE
Threading should be tested in Travis CI.

### ISSUE THREAD

**Issue by devinamatthews** [flame/blis#101]
It seems that commit 970745a introduced a bug which only manifests when threading is enabled (see #100). Build configurations for openmp and pthreads threading should be added to Travis CI to catch these kinds of bugs.

**Comment by jeffhammond** [comment]
And it's easy to test many versions of GCC in Travis CI Mac instances using Homebrew.  Obviously, one can install RPMs in Linux instances with sudo enabled, but that's not ideal for other reasons.

**Comment by fgvanzee** [comment]
Once, at a conference in Boston I think, Jeff taught me me the phrase, "We look forward to your patch." :)

I'm open to this, but I see it as a relatively low priority given how infrequently I dork with the multithreading functionality, and how I am usually quite thorough about testing those changes. (Granted, 970745a was an exception.)

**Comment by jeffhammond** [comment]
@fgvanzee You may look forward to my patch 😉 

I don't always compute, but when I do, it's in parallel 😆

**Comment by devinamatthews** [comment]
Fixed by #108.
